test/openAPI/features/applicant_tasks_taskid.feature

```diff
@@ -0,0 +1,22 @@
+@method=GET @endpoint=/tasks/{taskId}
+Feature: The API endpoint used to get task information by task id
+
+  @smoke @unit @positive
+  Scenario: Successfully get task information smoke type test
+
+    Given User wants to get task information
+    When GET request to get task is sent with given "81c4445c-bff6-11ed-afa1-0242ac120002" as taskId parameter
```

Comment:
There is no "81c4445c-bff6-11ed-afa1-0242ac120002" taskId in the test-data.json file. Please add it.
Also don't use the same ids in a different tests ;)